src/api/configure.ts

```diff
@@ -54,6 +66,14 @@ export function configure(options: {
     if (computedRequiresReaction !== undefined) {
         globalState.computedRequiresReaction = !!computedRequiresReaction
     }
+    if (reactionRequiresObservable !== undefined) {
+        globalState.reactionRequiresObservable = !!reactionRequiresObservable
+    }
+    if (observableRequiresReaction !== undefined) {
+        globalState.observableRequiresReaction = !!observableRequiresReaction
```

Comment:
if `observableRequiresReaction` is set, we might want to assert that some mode of `enforceActions` is also set?
It also implies `computedRequiresReaction` I guess

Reply:
It's true that `computedRequiresReaction` makes most sense in conjunction with `enforceAction: "observed/always"`
But strictly speaking isn't required... you can still perform writes `this.x = 5;` (without reads) outside of `action/reaction`. 

`computedRequiresReaction` works a bit differently as it has different purpose - it was introduced to prevent `cache` purging.
As a result it actually doesn't matter from where the `computed` is accessed as long as there is at least one observer. 
In contrast `observableRequiresReaction` makes the `observable` inaccessible outside `reaction`/`action`, no matter how many observers.
Currently `observableRequiresReaction` has no effect on `computed`, as there is no need to ... it will throw anyway on the first observable accessed from within the computed...

Reply:
Check, for the first one I hoped it would make error messages better, but now that I think of it, it wouldn't help if an non-annoted action would start of with a read, so probably we should add the possibliity that an action might be needed to the error message.

Agreed on the second one, it was more of an observation than anything else :)

Reply:
Is there any item from here?